modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/DLManagementToolbarPropsTransformer.js

```diff
@@ -231,6 +232,37 @@ export default function propsTransformer({
 		});
 	};
 
+	const permissions = () => {
+		const searchContainer = Liferay.SearchContainer.get(
+			otherProps.searchContainerId
+		);
+
+		if (searchContainer.select) {
+			const keys = searchContainer.select
+				.getAllSelectedElements()
+				.get('value');
+
+			if (keys) {
```

Comment:
I'm not sure if this safe condition is necessary. 

When we have a search container with a selection we can assume that we are going to receive something on `keys`, but maybe I missing something.

In case this is necessary I prefer to add a fallback value and avoid indentation:

```js
			const keys = searchContainer.select
				.getAllSelectedElements()
				.get('value') || [];
```

Reply:
What condition?
`if (searchContainer.select) {` ?
`if (keys) {` ?
both?